homeassistant/components/yeelight/light.py

```diff
@@ -281,6 +300,42 @@ def light_type(self):
         """Return light type."""
         return self._light_type
 
+    @property
+    def hs_color(self) -> tuple:
+        """Return the color property."""
+        return self._hs
+
+    # F821: https://github.com/PyCQA/pyflakes/issues/373
+    @property
+    def _bulb(self) -> 'yeelight.Bulb':  # noqa: F821
```

Comment:
As imports can be done on the top level, this could be converted to simply `Bulb`, right?